pkgs/by-name/ca/cargo-3ds/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-3ds";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
```

Comment:
Since the package is pulled from GitHub with proper releases, could we think about adding an auto-update script such as `nix-update` or similar (if the script works correctly for this project, that is):
```nix
passthru.updateScript = nix-update-script { };
```

Reply:
Would it work just by adding that one line?

Reply:
Hopefully is will. But you should check that it indeed does. For example, but specifying an older version and allowing the update script, through running `nix run nixpkgs#nix-update -- cargo-3ds`, and see that it used the update script and updated the package to 0.1.3.

Reply:
I've ran the command, and confirmed it works. Thank you!